### PR TITLE
ai dorf translation (and other races)

### PR DESCRIPTION
**Pull request by scarwolff**
## About The Pull Request

AI's and robots can now understand dwarf speak (they cannot speak it though because its magical rune language ooooh ahhh ooh), also makes a language icon folder and adds a mini dwarf icon for when they speak dwarvish. Edit: Adds in the other races and mini-icons too, AIs can now understand spider and squiddish, but can only speak squiddish.

image

image

image

## Why It's Good For The Game

Silicons cant understand dwarves and the dwarf language is like the "clear version" of what they talk in a way. Makes it easier for giving orders to them if they dont understand you. EDIT: Also adds it for the other two races we got too, the change being they can speak squiddish because its a noise instead o' magic or movements or shummin.

## Changelog
:cl:
add: AI's can now understand dwarves (and other WS races)
add: Makes a white sands language icon folder
add: Silicons can speak squid now too
tweak: Dwarven language now has a mini icon (and other WS races)
/:cl:

### CODE REVIEW

**Comment by tiramisuapimancer** [comment]
they ought to have the personal languages of all station sideds yea ?

**Comment by scarwolff** [comment]
> they ought to have the personal languages of all station sideds yea ?

done

**Comment by Shadowtail117** [comment]
> add: Makes a white sands language icon folder

No need to make an entry for exclusively backend changes.

**Comment by Maldaris** [comment]
> > add: Makes a white sands language icon folder
> 
> No need to make an entry for exclusively backend changes.

Nah this is correct

File: code/modules/language/language_holder.dm [hunk 1]
```diff
@@ -310,13 +310,17 @@ Key procs
...
+								/datum/language/spider = list(LANGUAGE_ATOM), //WS Edit - Spider Translation
+								/datum/language/dwarf = list(LANGUAGE_ATOM)) //WS Edit - Dwarf Translation
 	spoken_languages = list(/datum/language/common = list(LANGUAGE_ATOM),
 							/datum/language/machine = list(LANGUAGE_ATOM),
 							/datum/language/draconic = list(LANGUAGE_ATOM),
 							/datum/language/moffic = list(LANGUAGE_ATOM),
 							/datum/language/calcic = list(LANGUAGE_ATOM),
-							/datum/language/voltaic = list(LANGUAGE_ATOM))
+							/datum/language/voltaic = list(LANGUAGE_ATOM),
+							/datum/language/rylethian = list(LANGUAGE_ATOM)) //WS Edit - Squid Translation
```
Comment (Maldaris) [inline code/modules/language/language_holder.dm:323]: You have two of these defined.

**Review by Maldaris — CHANGES_REQUESTED** [review]

File: code/modules/language/language_holder.dm (hunk 1 above)
Comment (scarwolff) [reply code/modules/language/language_holder.dm:323]: one for hearing and ones for speaking

**Review by Maldaris — APPROVED** [review]

**Pull request merged**